parquet-avro/src/test/java/org/apache/parquet/avro/TestSpecificReadWrite.java

```diff
@@ -237,6 +242,43 @@ public void testAvroReadSchema() throws IOException {
     }
   }
 
+  @Test
```

Comment:
We need to test both of `getModelForSchema` related to the avro version. If the version check gets more complicated, maybe more versions are to cover.